### ISSUE TITLE
Repo out of date vs Nuget?

### ISSUE THREAD

**Issue by SteveK-GS** [sendgrid/smtpapi-csharp#21]
tl;dr: It looks like the code here is a version behind Nuget.

I've cloned the main C# libraries (as I prefer to see the code when it's available) but https://github.com/sendgrid/sendgrid-csharp, when referencing https://github.com/sendgrid/smtpapi-csharp instead of the smtapi Nuget package, fails to build with these errors:

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1061	'IHeader' does not contain a definition for 'SetSendEachAt' and no extension method 'SetSendEachAt' accepting a first argument of type 'IHeader' could be found (are you missing a using directive or an assembly reference?)	Mail	D:\FleetTrackProjects\Open Source\SendGrid-csharp\SendGrid\SendGridMail\SendGrid.cs	264	Active

Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1061	'IHeader' does not contain a definition for 'SetSendAt' and no extension method 'SetSendAt' accepting a first argument of type 'IHeader' could be found (are you missing a using directive or an assembly reference?)	Mail	D:\FleetTrackProjects\Open Source\SendGrid-csharp\SendGrid\SendGridMail\SendGrid.cs	259	Active

The problem appears to be that smtapi NuGet is version 1.3.1, but the version in this repository is 1.3.0. So, where is the source code for smtapi-csharp v1.3.1?

I've resolved for now by adding the following to IHeader:

void SetSendAt(DateTime sendTime);
void SetSendEachAt(IEnumerable<DateTime> sendDateTimes);

**Comment by thinkingserious** [comment]
This is the latest source code for this library.

I'm not sure what's going wrong with the versioning, but the new library (coming very shortly) that supports our v3 Web API will not have this issue.

Thanks for taking the time to report this error and to provide a fix. 

If you don't mind sending us your T-shirt size and mailing address, we'd be happy to swag you out. 

dx at sendgrid.com
